### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'componentConfig')

### DISCUSSION THREAD

**Discussion by hackun666** [NervJS/taro#11353] [Q & A]
Module build failed (from ./node_modules/_vue-loader@15.9.3@vue-loader/lib/loaders/templateLoader.js):
TypeError: Cannot read properties of undefined (reading 'componentConfig')

anyone who can help me with this error,thx

**Comment by IsabellaLiYue** [comment]
<img width="943" alt="image" src="https://user-images.githubusercontent.com/38759280/158348907-a7bfbde9-0f72-436b-b46a-25fee448c9a2.png">
尝试改下taro-cli 版本，我和你问题一样，我改了版本，成功了。
